test/unit/migrations/048-test.js

```diff
@@ -237,8 +237,8 @@ describe('migration #48', function () {
 
     const newStorage = await migration48.migrate(oldStorage)
     assert.deepEqual(
-      { ...oldStorage.data, ...expectedPreferencesState },
-      { ...newStorage.data, ...expectedPreferencesState },
+      { ...expectedPreferencesState, ...oldStorage.data },
+      { ...expectedPreferencesState, ...newStorage.data },
```

Comment:
It's fine either way, but this is nevertheless better. As pointed out here: https://github.com/MetaMask/metamask-extension/pull/9551#discussion_r503095080

cc: @kumavis